infogami/infobase/__init__.py

```diff
@@ -1,10 +1,14 @@
 """
 Infobase.
 """
+from __future__ import print_function
+
+import logging
 import sys
+
 import web
 
-import config, infobase, logger, logreader
+from infogami.infobase import config, infobase, logreader
```

Comment:
logreader unused as well

Reply:
resolving per the flake8 evidence above not showing these as unsed..